src/util/dev/server.ts

```diff
@@ -329,11 +347,70 @@ export default class DevServer {
       }
     }
 
+    const apiFiles = await getApiFiles(this.cwd, this.output);
+
+    if (apiFiles.length > 0) {
+      if (!config.builds || config.builds.length === 0) {
+        config.builds = [];
+
+        if (pkg) {
+          const staticBuilder = await detectBuilder(pkg);
+
+          if (staticBuilder) {
+            config.builds.push(staticBuilder);
+          }
+        }
+
+        if (apiFiles && apiFiles.length > 0) {
+          const apiBuilds = await detectApiBuilders(apiFiles);
```

Comment:
Does `detectApiBuilders()` actually return `null` or is it always an empty array? I'm asking because of the `if (apiBuilds && apiBuilds.length > 0) {` check below (IMO it would be cleaner to always return an empty array, unless the falsey case means something semantically different).

Reply:
@TooTallNate it will return `null` to make sure it uses `@now/static` instead. Basically it won't make `.builds` an array it it wasn't one before and did not return any builds.

The reason is because the function is used by our APIs and they need it this way.